Library/Formula/fstar.rb

```diff
@@ -0,0 +1,92 @@
+class Fstar < Formula
+  desc "Language with a type system for program verification"
+  homepage "https://www.fstar-lang.org/"
+  url "https://github.com/FStarLang/FStar.git",
+    :tag => "v0.9.1.1",
+    :revision => "c41ea17778ff2fef4e75d56604ce1804e1985f33"
+  head "https://github.com/FStarLang/FStar.git"
+
+  depends_on "mono"   => :build
+  depends_on "opam"   => :build
+  depends_on "ocaml"  => :recommended
+  depends_on "z3"     => :recommended
+
+  resource "ocamlfind" do
+    url "http://download.camlcity.org/download/findlib-1.5.5.tar.gz"
+    sha256 "aafaba4f7453c38347ff5269c6fd4f4c243ae2bceeeb5e10b9dab89329905946"
+  end
+
+  resource "batteries" do
+    url "https://github.com/ocaml-batteries-team/batteries-included/archive/v2.3.1.tar.gz"
+    sha256 "df778b90fcdb26288d9d92a86e51dd75d6bb7c6e41888c748c7508e8ea58b1d4"
+  end
+
+  def install
+    ENV.deparallelize
```

Comment:
Ah of course. Remember this from elsewhere now. Thanks for kicking it to the right upstream. It may be worth adding a comment in-formula that the line isn't an `fstar` problem.